cli/connector/binary.go

```diff
@@ -0,0 +1,85 @@
+package connector
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/dokshina/tarantool"
```

Comment:
This is not the same as what is written in commit message.

"For binary protocol: https://github.com/FZambia/tarantool". But here you use your fork without any comments why do you do that. Finally this switch could be moved into separate commit (https://github.com/tarantool/cartridge-cli/compare/56bffec679eadda4c76424bb24ee9202f45915a3..ea5ef182ccd1d2958c57e069de3541f7089e0395)

Reply:
See https://github.com/tarantool/cartridge-cli/issues/452

I suggest merging this patch as is (without using github.com/dokshina/tarantool). After merging my patch into connector upstream I will fix calling `ExecTypedContext` in a a separated patch.